LICENSE.md

```diff
@@ -12,13 +12,13 @@ Version 2.0.2
 
 **User**: Anyone who that interacts with Minecraft or the Mod in any way.
 
-**Dependency**: Software required for the mod to execute, compile, or otherwise "work" correctly.
+**Dependency**: Software required for the Mod to execute, compile, or otherwise "work" correctly.
 
 **API**: Software intermediary for interaction with the Mod.
 
-**Derived Code**: Software which includes any portion of the Mod other than the API, with modifications, additions, or subtractions.
+**Derivative**: Software which includes any portion of the Mod other than the API, with modifications, additions, or subtractions.
 
-**Addon**: Software that extends on this mod, but is not a derivative.
+**Addon**: Software that extends on the Mod, but is not a derivative.
```

Comment:
Could probably remove "on" eg "extends the Mod", but that's a small nitpicky thing.